hydrosdk/modelversion.py

```diff
@@ -513,89 +521,6 @@ def __repr__(self):
         return "Model {}:{}".format(self.name, self.version)
```

Comment:
I think there should be ModelVersion
```suggestion
        return "ModelVersion {}:{}".format(self.name, self.version)
```